eventmesh-meta/eventmesh-meta-etcd/src/main/java/org/apache/eventmesh/meta/etcd/service/EtcdCustomService.java

```diff
@@ -67,6 +67,9 @@ public List<EventMeshServicePubTopicInfo> findEventMeshServicePubTopicInfos() th
                 return eventMeshServicePubTopicInfoList;
             }
         } catch (Exception e) {
+            if (e instanceof InterruptedException) {
+                Thread.currentThread().interrupt();
+            }
             log.error("[EtcdRegistryService][findEventMeshServicePubTopicInfos] error", e);
             throw new MetaException(e.getMessage());
         }
```

Comment:
Please catch `InterruptedException` in a separate `catch` code block instead of judging it. You may see `org.apache.eventmesh.meta.etcd.service.EtcdMetaService` L173-175 and L300-302 for examples.

https://github.com/apache/eventmesh/blob/21e4c3a36816576de7eec903c885cf6e5f956c31/eventmesh-meta/eventmesh-meta-etcd/src/main/java/org/apache/eventmesh/meta/etcd/service/EtcdMetaService.java#L173-L179

https://github.com/apache/eventmesh/blob/21e4c3a36816576de7eec903c885cf6e5f956c31/eventmesh-meta/eventmesh-meta-etcd/src/main/java/org/apache/eventmesh/meta/etcd/service/EtcdMetaService.java#L298-L305

Reply:
get